janekv20/find-my-boards
# Feature/19 profile picture

What Branch Does:

closes #19 
closes #27 

Adds a profile handlebars template for the profile pic upload form
created profile-routes file to render the profile hb template & add functionality to save the profile pic file to the 'profile-pics' folder and also update the image key for the specific user in our database

How to test:
navigate to http://localhost:3001/profile/88 or whatever id you'd like. Upload a photo. Make sure the following happens:

1) the picture you uploaded gets saved in the profile-pics folder in our project
2) in insomnia, run a get user route on the id you updated and make sure the image key value is the name of the file you uploaded
3) you are redirected back to the same profile webpage http://localhost:3001/profile/88 or whatever id you used
